### DISCUSSION TITLE
Use Langchain LCEL + Langserve + run time user_id/session_id

### DISCUSSION THREAD

**Discussion by ravediamond** [langfuse/langfuse#2501] [Ideas]
### Describe the feature or potential improvement

When integrating Lanfuse with LCEL, for example in the context of an api with Langserve, it is not possible to set at runtime the `user_id` or the `session_id`. 
I propose to check the Langchain configurable if there are variables called `user_id` or `session_id` and use them to populate these fields in Langfuse.
This is actually what I do in my LangServe API and it is working great.

What do you think about adding this in official callback for Langchain ?

**Comment by marcklingen** [comment]
Hi @ravediamond, open for contributions on this, this is a great idea!
